packages/kyt-cli/cli/actions/setup.js

```diff
@@ -414,10 +414,25 @@ module.exports = (flags, args) => {
     removeTmpRepo();
 
     if (localPath) {
-      shell.cp('-r', localPath, tmpRepo);
-      afterClone();
+      shell.exec(`cp -R ${localPath} ${tmpRepo}`);
+      afterCopy();
+    } else if (npmName) {
+      shell.mkdir(tmpRepo);
+      shell.cd(tmpRepo);
+      const fakePkgPath = `${tmpRepo}/package.json`;
+      fs.writeFileSync(fakePkgPath, JSON.stringify(fakePackageJson, null, 2));
+      let iCmd = 'npm install';
```

Comment:
hmm i think it would probably just work if you supplied an `npmName` like `foo@0.1.0`, right?

related: can we make it so that you can supply an `npmName` on the command line?